src/recon_test/recontest.cxx

```diff
@@ -71,24 +70,21 @@ int main(int argc, const char *argv[])
     shared_ptr < Reconstruction < DiscretisedDensity < 3, float > > >
             reconstruction_method_sptr;
 
-    std::string output_filename;
-
     KeyParser parser;
     parser.add_start_key("Reconstruction");
     parser.add_stop_key("End Reconstruction");
-    parser.add_key("output filename prefix", &output_filename);
     parser.add_parsing_key("reconstruction method", &reconstruction_method_sptr);
     parser.parse(argv[1]);
 
     HighResWallClockTimer t;
     t.reset();
     t.start();
 
-
     if (reconstruction_method_sptr->reconstruct() == Succeeded::yes)
     {
         t.stop();
         std::cout << "Total Wall clock time: " << t.value() << " seconds" << std::endl;
+        return Succeeded::yes;
```

Comment:
a `main()` function has to return `EXIT_SUCCESS`

Reply:
I will correct this when we see what will happen with the rest code